src/pages/sourcecast/Sourcecast.tsx

```diff
@@ -275,6 +283,26 @@ class Sourcecast extends React.Component<SourcecastProps> {
   }
 }
 
+const parseSourcecastUID = (props: SourcecastProps) => {
```

Comment:
React has a router you know. 

https://reactrouter.com/web/example/url-params

Reply:
Okay will fix it.